pandas/tests/reshape/test_concat.py

```diff
@@ -2832,3 +2832,17 @@ def test_concat_preserves_subclass(obj):
 
     result = concat([obj, obj])
     assert isinstance(result, type(obj))
+
+
+def test_concat_frame_axis0_extension_dtypes():
```

Comment:
This feels pretty specific to Int64, so I might have placed in in `pandas/tests/arrays/integer`, but this works too.

Reply:
this is testing pd.concat directly; i think this is a reasonable place for this test.

Reply:
Yes, it is testing specifically that EAs in frames take the correct concat code path. I could also have used another EA such boolean or string. So this is frame specific.

The actual concat logic (Int64 + int64 -> Int64, Int64 + float64 -> float64, etc) is tested in `tests/arrays/integer/test_concat.py`, and doesn't need to involve frames in principle, as long as we test elsewhere (here) that frames take the EA concatting code path.